battleship/Form1.cs

```diff
@@ -12,9 +12,26 @@ namespace battleship
 {
     public partial class Form1 : Form
     {
+        WMPLib.WindowsMediaPlayer musicPlayer = new WMPLib.WindowsMediaPlayer();
         public Form1()
         {
             InitializeComponent();
         }
+
+        private void Form1_Load(object sender, EventArgs e)
+        {
+            musicPlayer.URL = (@"StartMusic.mp3");
+            musicPlayer.settings.volume = 15;
+            musicPlayer.controls.play();
+        }
+
+        private void beginButton_Click(object sender, EventArgs e)
+        {
+            startScreen.Enabled = false;
+            startScreen.Visible = false;
+            beginButton.Enabled = false;
+            beginButton.Visible = false;
+            musicPlayer.controls.stop();
```

Comment:
When the start screen pops up you mean?
On Nov 10, 2015 10:54 AM, "Ontrox" <notifications@github.com> wrote:

> In battleship/Form1.cs
> <https://github.com/ninoaguilar/Battleship/pull/23#discussion_r44439959>:
>
> >          public Form1()
> >          {
> >              InitializeComponent();
> >          }
> > +
> > +        private void Form1_Load(object sender, EventArgs e)
> > +        {
> > +            musicPlayer.URL = (@"StartMusic.mp3");
> > +            musicPlayer.settings.volume = 15;
> > +            musicPlayer.controls.play();
> > +        }
> > +
> > +        private void beginButton_Click(object sender, EventArgs e)
> > +        {
> > +            startScreen.Enabled = false;
> > +            startScreen.Visible = false;
> > +            beginButton.Enabled = false;
> > +            beginButton.Visible = false;
> > +            musicPlayer.controls.stop();
>
> Maybe we should play a fanfare or something here, but that is totally
> optional.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ninoaguilar/Battleship/pull/23/files#r44439959>.
>